flag_engine/features/schemas.py

```diff
@@ -68,6 +68,9 @@ def serialize_value(self, instance: object) -> typing.Any:
         if isinstance(instance, dict) and "value" in instance:
             return instance["value"]
 
+        if isinstance(instance, FeatureStateModel):
```

Comment:
@matthewelwell  not sure about this one, should we implement get_feature_state_value?

Reply:
Interesting. Yes, perhaps for now, let's just add something like: 

```python
def get_feature_state_value(self):
    """Mimick django method name to simplify serialization logic"""
    return self.get_value()
```